### ISSUE TITLE
admob.initialize() causes crash on older iOS

### ISSUE THREAD

**Issue by jm1175** [defold/extension-admob#15]
The app crashes when calling admob.initialize() on an iPhone 6 Plus running iOS 12.5.4. It works fine on an iPhone 12 running iOS 14.7.1.

**Comment by JCash** [comment]
Thanks for the report.
Any logs you can share?

**Comment by jm1175** [comment]
Yes this is from the log file:
```
INFO:DLIB: Writing log to: /var/mobile/Containers/Data/Application/484D32D0-5D3D-4C0F-AF9A-922DA578F76D/Documents/log.txt
INFO:ENGINE: Loading data from: dmanif:/var/containers/Bundle/Application/D3461A7E-2ECF-414C-B20A-BC153FE72AC7/Tricky Bounce.app/game.dmanifest
INFO:GpgsExt: Registered extension Gpgs (null)
INFO:InstantApp: Registered extension InstantApp (null)
INFO:ENGINE: Initialised sound device 'default'
DEBUG:SCRIPT: save successful
DEBUG:SCRIPT: Using Admob test unit ID
DEBUG:SCRIPT: EVENT_NOT_SUPPORTED: IDFA request not supported on this platform or OS version
INFO:DLIB: SSDP: Started on address 10.0.0.4
INFO:DLIB: SSDP: Started on address 169.254.123.97
INFO:CRASH: Successfully wrote Crashdump to file: /var/mobile/Containers/Data/Application/484D32D0-5D3D-4C0F-AF9A-922DA578F76D/Library/Application Support/Defold/_crash
ERROR:CRASH: CALL STACK:


ERROR:CRASH: 0   TrickyBounce                        0x00000001011d1e2c _ZN7dmCrash7OnCrashEi + 104

ERROR:CRASH: 1   libsystem_platform.dylib            0x00000001af07a9fc <redacted> + 56

ERROR:CRASH: 2   TrickyBounce                        0x0000000100ede99c GADDeviceAudioPortNames + 52

ERROR:CRASH: 3   TrickyBounce                        0x0000000100eded34 GADDeviceAudioPortNames + 972

ERROR:CRASH: 4   TrickyBounce                        0x0000000100edece8 GADDeviceAudioPortNames + 896

ERROR:CRASH: 5   TrickyBounce                        0x0000000100f7c948 GAD_GADSignals_arm64_8_5_0 + 4380

ERROR:CRASH: 6   libdispatch.dylib                   0x00000001aeea8a38 <redacted> + 24

ERROR:CRASH: 7   libdispatch.dylib                   0x00000001aeea97d4 <redacted> + 16

ERROR:CRASH: 8   libdispatch.dylib                   0x00000001aee57008 <redacted> + 1068

ERROR:CRASH: 9   CoreFoundation                      0x00000001af3fcb20 <redacted> + 12

ERROR:CRASH: 10  CoreFoundation                      0x00000001af3f7a58 <redacted> + 1924

ERROR:CRASH: 11  CoreFoundation                      0x00000001af3f6fb4 CFRunLoopRunSpecific + 436

ERROR:CRASH: 12  GraphicsServices                    0x00000001b15f879c GSEventRunModal + 104

ERROR:CRASH: 13  UIKitCore                           0x00000001dbe42c38 UIApplicationMain + 212

ERROR:CRASH: 14  TrickyBounce                        0x0000000100fdfd90 _ZN8dmEngine7RunLoopEPKNS_13RunLoopParamsE + 44

ERROR:CRASH: 15  TrickyBounce                        0x0000000100fdfd50 _Z11engine_mainiPPc + 96

ERROR:CRASH: 16  libdyld.dylib                       0x00000001aeeba8e0 <redacted> + 4

ERROR:CRASH: 

```

**Comment by AGulev** [comment]
@jm1175 should be fixed in Defold 1.2.188+  and AdMob 2.0.4+

**Comment by jm1175** [comment]
Yes it's working, thanks! 👍
